scripts/docker-entrypoint-node-1.sh

```diff
@@ -1,17 +1,18 @@
 #!/bin/bash
 
+# FIXME - check security associated with each CLI option
 ../target/release/datahighway --validator \
   --unsafe-ws-external \
   --unsafe-rpc-external \
   --rpc-cors=all \
   --base-path /tmp/polkadot-chains/node-1 \
-  --bootnodes /dns4/testnet-frankfurt-v0.1.0-alpha.datahighway.com/tcp/30333/p2p/QmWYmZrHFPkgX8PgMgUpHJsK6Q6vWbeVXrKhciunJdRvKZ \
+  --bootnodes /dns4/${ENDPOINT_TESTNET}/tcp/30333/p2p/${SENTRY_NODE_TESTNET_1} \
   --keystore-path "/tmp/polkadot-chains/node-1/keys" \
-  --chain ../src/chain-definition-custom/chain_def_testnet_v0.1.0.json \
-  --name "Example Validator Node 1" \
+  --chain ../src/chain-definition-custom/chain_def_${CHAIN_VERSION}.json \
+  --name "${NODE_ENV} Validator Node 1" \
   --port 30333 \
   --ws-port 9944 \
   --rpc-port 9933 \
-  --telemetry-url ws://telemetry.polkadot.io:1024 \
+  --telemetry-url disabled \
```

Comment:
Any reason to turn off telemetry for all nodes?

Reply:
i've changed it so all the validators appear in telemetry again